src/lib/Importer/SchemaImporter.php

```diff
@@ -176,6 +181,48 @@ private function ensureNoExtraKeys(array $tableConfiguration, string $location,
             ));
         }
     }
+
+    /**
+     * @param string|array<string, mixed> $indexConfig
+     *
+     * @throws \Ibexa\Contracts\DoctrineSchema\Exception\InvalidConfigurationException
+     */
+    private function addIndexToColumn($indexConfig, string $location, Table $table, string $columnName): void
```

Comment:
I don't agree with union here. This should be strict - either array or string. In other words, if you want to allow simplification, it needs to be handled before passing config to this method.